### DISCUSSION TITLE
In the Marketplace, how would you onboard new users with the free plan?

### DISCUSSION THREAD

**Discussion by rafasashi** [community/community#37592] [Other Feature Feedback, Questions, & Ideas]
Quick question regarding the onboarding process of an app via the Marketplace.

Since the purpose of the app is to import GitHub repos into our service we left it free and implemented the onboarding that way:

1 - the user is redirected from our dashboard to the GitHub install page 
2 - the user selects the account and is redirected to the callback url
3 - we use a "state" token to match the install id against the logged user

How would you onboard new users with this flow from the free plan of the Marketplace?

**Comment by ossanna16** [comment]
Hi there @rafasashi and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by rafasashi** [reply]
Hi Anna,

Thank you for the warm welcome and insights about the community.

To nurture your curiosity here is [my introduction](https://github.com/orgs/community/discussions/23204?sort=new#discussioncomment-4024235) :smile:
